### ISSUE TITLE
Documentation small additions/changes

### ISSUE THREAD

**Issue by RuslanBergenov** [adswerve/google_analytics_flattener_ga4#13]
1) say that they need to manually add a label to staging code GCS bucket, in order to track costs
```
key: bucket
value: ga4flattener-function-code-staging
```

2) 
move this paragraph up, into item 4:
If your GCP project is brand new, you might not have [PROJECT_NUMBER]@cloudservices.gserviceaccount.com yet. To fix this, enable Compute Engine API and then disable it. The service account [PROJECT_NUMBER] @cloudservices.gserviceaccount.com will appear in your GCP project under IAM.

3) 
Maybe add a list of shell commands used for installation?

TLDR:
```
# install
gcloud init
gcloud config configurations list
gcloud config configurations activate {configuration}
gcloud deployment-manager deployments create ga4-flattener-deployment --config ga_flattener.yaml

# describe
gcloud deployment-manager deployments describe ga4-flattener-deployment

# uninstall
gsutil rm gs://ga4-flattener-deployment-{project_number}-adswerve-ga-flat-config/config_datasets.json
gcloud deployment-manager deployments delete ga4-flattener-deployment -q
```

4) Put a disclaimer about the intraday feature - user properties are not accurate in the intraday table. This is related to this issue:
https://github.com/adswerve/google_analytics_flattener_ga4/issues/12

5) Explain this in README. this is related to this issue: https://github.com/adswerve/google_analytics_flattener_ga4/issues/9 

RE: how to edit the config file

If you want non-default config file options, you need to do the following:

- manually download the config file from GCS (Google Cloud Storage)
- Edit it locally in a text/code upload
- Upload the config file back to GCS and overwrite the old version.
- We will make sure to add these steps to README.

**Comment by RuslanBergenov** [comment]
Addressed in this branch: https://github.com/adswerve/google_analytics_flattener_ga4/tree/development-readme-updates

**Comment by RuslanBergenov** [comment]
Fixed and released in vReleased in v1.2.1.
